mitosheet/src/mito/components/taskpanes/Graph/graphUtils.tsx

```diff
@@ -31,6 +31,9 @@ export interface GraphElementType {
         top?: number,
         bottom?: number,
     }
+
+    /** Whether to display the popup */
```

Comment:
This comment isn't really helpful. Can you describe the three cases, what they are, and how a user would get there? Just a short sentence each.